lib/galaxy/datatypes/binary.py

```diff
@@ -2770,6 +2770,28 @@ def sniff(self, dataset):
         return False
 
 
+@build_sniff_from_prefix
+class Parquet(Binary):
+    """
+    Class describing Apache Parquet file (https://parquet.apache.org/)
+    >>> from galaxy.datatypes.sniff import get_test_fname
+    >>> fname = get_test_fname('example.parquet')
+    >>> Parquet().sniff(fname)
+    True
+    >>> fname = get_test_fname('test.mz5')
+    >>> Parquet().sniff(fname)
+    False
+    """
+    file_ext = "parquet"
```

Comment:
Is there an EDAM datatype for it?

Reply:
I had a brief look but it seems generic to me in all scopes. Do you think it fits somewhere?

Reply:
afaik Parquet is a csv on steroids, I wouldn't know how to edam-tag csv either

Reply:
I couldn't find a specific one either. If you like, you can request it to be added at https://github.com/edamontology/edamontology/issues

Reply:
If we succeed to enough extent to apply parquet in the mass spectrometry pipelines we'll consider PRing it to EDAM, at the moment I think it is a bit premature.